activity/src/test/java/com/amazonaws/stepfunctions/cloudformation/activity/DeleteHandlerTest.java

```diff
@@ -70,7 +83,9 @@ public void test500() {
 
     @Test
     public void testUnknownException() {
-        Mockito.when(proxy.injectCredentialsAndInvoke(Mockito.any(DeleteActivityRequest.class), Mockito.any(Function.class))).thenThrow(unknownException);
+        Mockito.when(proxy.injectCredentialsAndInvoke(Mockito.any(), Mockito.any(Function.class)))
+                .thenReturn(new DescribeActivityResult())
+                .thenThrow(unknownException);
```

Comment:
an unknown exception is also a `500` right? or do we not set a status code when it's unknown?

similar to earlier nit: maybe rename the method names to better reflect the scenario under test.